ct/cmd/root.go

```diff
@@ -94,4 +94,5 @@ func addCommonLintAndInstallFlags(flags *pflag.FlagSet) {
 	flags.Bool("debug", false, heredoc.Doc(`
 		Print CLI calls of external tools to stdout (Note: depending on helm-extra-args
 		passed, this may reveal sensitive data)`))
+	flags.Bool("print-config", true, "Prints the configuration to stderr")
```

Comment:
This flag seems a bit not intuitive to me. The default is to print the config (this has a true value). So, you don't pass in the flag to skip it. You have to say it's equal to false. Would it be better to make this `--skip-print-config` with a value of false? Then use the `not` of this value for the `LoadConfiguration` function?

Reply:
Good point. not sure what's better. Ideally, I'd like to change the default to false. I was hesitant to do so. Would that be considered a breaking change?

Reply:
Changing a default value is considered a breaking change. The helm client would only do that on a major version change.

Reply:
This is fun, because we already have a `printConfig` arg in `LoadConfiguration()`, but had no actual config/flag for it 😄 

Given the security implications I think we should change chart-testing to print not the config values by default, and whatever flag we decide is most intuitive to expose to end uses add a note in the flag help/description what you said initially @unguiculus:

> caution: setting this may expose sensitive data when helm-repo-extra-args contains passwords

I'm a bit torn on the semver interpretation here. If it's considered a security bug, then changing output wouldn't require MAJOR. In that case, the additional flag to output it if desired would only require MINOR. Thoughts on this?

Reply:
I agree with @scottrigby. After all it's only a change in log output, not in functionality.

Reply:
Cool. Whatever we decide with semver, not printing this by default is the right thing to do